app/pods/publications/publication/proofs/documents/template.hbs

```diff
@@ -159,6 +159,7 @@
 {{#if this.isPieceUploadModalOpen}}
   <Publications::Publication::Proofs::ProofUploadModal
     @publicationFlow={{this.publicationFlow}}
+    @source="correction"
```

Comment:
Alle documenten die je upload in de drukproeven / documenten view is een correctie?
Ik ken de flow van drukproeven niet goed, maar wil je hier geen andere documenten uploaden (vb een bron document voor het aanvragen van een drukproef?

Ook geven we de source hier hardcoded mee op 2 plaatsen.
Maar met de waarde "correction" wordt verder niks gedaan. enkel "proof" heeft een gevolg
Willen we dan werken met een boolean ipv tekst? of met Constants als je toch tekst wilt

Reply:
Op je eerste comment: dit klopt. Ze kunnen enkel correcties opladen op de drukproeven/documenten-tab. De 'brondocumenten' worden opgeladen bij de vertalingen en worden daar gemarkeerd als 'voor drukproef'. Die verschijnen dan automatisch op de drukproeven/documenten-tab. Ik zal het label van de button aanpassen naar 'Verbeterde drukproef opladen' om verwarring te vermijden.

Reply:
Je hebt wel gelijk dat we met `@source="correction"` en `@source="proof"` twee concepten aan het mixen zijn.

Ik denk dat de flags die we definieren op de upload modal component beter zijn:
- `isReceivedDocument` (`true` of `false`): indien `true` 
  - tonen we het invulveld voor ontvangstdatum
- `isCorrectionDocument` (`true` of `false`): indien `true` 
  - tonen we als titel van de modal 'Verbeterd drukproef uploaden voor publicatie...'
  - een read-only veldje na het uploaden van de file 'Verbeterd' met het groene vinkje zoals in de tabel (dus zoals de verbeterd-toggle in het prototype, maar dan met een read-only vinkje ipv de toggle). Op die manier is het waarschijnlijk duidelijker voor de gebruiker dat ze een verbeterd document aan het opladen zijn. Anders zien ze het verbeterd-vinkje pas verschijnen na het opladen in de tabel.

Reply:
Opgelost in https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/1006/commits/17cc0fb19e3a9e39267758a178470f13c11fa706 en https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/1006/commits/79d3168dccd939cd48f89ad7aa6f4112b248f5ac